envisage/tests/test_ids.py

```diff
@@ -0,0 +1,61 @@
+# (C) Copyright 2007-2019 Enthought, Inc., Austin, TX
```

Comment:
Copyright end year.

Reply:
i intentionally left this out because i wanted to update all of the copyright headers in envisage at once in a separate PR - https://github.com/enthought/envisage/issues/264